helsingborg-stad/aterbruket-webapp
# Ad borrow date range

User can now define a date range when creating an ad.
This PR only sets allowed borrow dates in DB, a page in front end with the ad and availed dates where the item can be borrows is missing currently.

## Testing
1) `Amplify push` needed as date range will be added to object in data base.
2) Create new ad (Utlåning) and define start and and range dates with calendar (Tillgänglig period).
3) Fill all required ad fields and create ad.
4) Check back end for created date range. New object "advertBorrowCalendar" should exist in database with defined allowedDateStart and allowedDateEnd.

## advertBorrowCalendar DB object
This object currently have two dates defined, allowedDateStart and allowedDateEnd.
These can be of types null or AWSDate.
allowedDateStart <null>: Will allow item to be borrowed from current date.
allowedDateEnd <null>: Will allow item to be borrowed to infinity and beyond or as long as the ad exists.
allowedDateStart date>: Will allow item to be borrowed from <date>.
allowedDateEnd <date>: Will allow item to be borrowed until <date>.
